pkg/processor/reportprocessor.go

```diff
@@ -191,68 +191,105 @@ func getGovernanceResults(dynamicClient dynamic.Interface, clusterInfo types.Man
 	}
 
 	var clusterViolations []*v1alpha2.PolicyReportResult
+
+	// Iterate over policies in the cluster
 	for _, plc := range policyList.Items {
-		plcName := plc.Object["metadata"].(map[string]interface{})["name"].(string)
-		func() {
-			defer func() {
-				if err := recover(); err != nil {
-					glog.V(2).Infof(
-						"Error processing policy %s - expected missing data to be present",
-						plcName,
-					)
+		// Parse relevant fields
+		plcName := plc.GetName()
+		md, _, err := unstructured.NestedMap(plc.Object, "metadata")
+		glog.Warningf("error parsing metadata as a map for policy %s: %s", plcName, err)
```

Comment:
Should there be a conditional of `if err != nil` here?

Reply:
Good point--I guess I was thinking not having "CreationTimestamp" was acceptable risk, but it should probably return early rather than having a partial update. The code is also running `GetAnnotations()` each time when it should be up here instead, so I'll move that also.